pyravendb/connection/requests_executor.py

```diff
@@ -39,7 +39,11 @@ def initialize_certificate(certificate):
             return certificate, None
         pfx = certificate["pfx"]
         password = certificate.get("password", None)
-        adapter = Pkcs12Adapter(pkcs12_data=pfx, pkcs12_password=password)
+        adapter = (
+            Pkcs12Adapter(pkcs12_filename=pfx, pkcs12_password=password)
+            if os.path.isfile(pfx)
```

Comment:
Shouldn't `if` be one line up?

Reply:
Nope, it's Python [tenary](https://book.pythontips.com/en/latest/ternary_operators.html)